app/views/shared/_contextual_header.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= render 'shared/login_banner' %>
+<%= render partial: 'shared/back_to_top', locals: {mobile: false} %>
+<%= render partial: 'shared/breadcrumbs', locals: {page_title: page_title, parent_pages: parent_pages} %>
+<%= render partial: 'shared/back_to_top', locals: {mobile: true} %>
```

Comment:
I struggled for a long time with CSS and couldn't figure out how to make this work without having 2 back to top button, one for the mobile design (below the breadcrumbs banner), and one for the desktop design (floating to the right and inline with the breadcrumbs, which are centered on the page. Open to changing this if someone knows how.

Reply:
Let’s take a look st this together when I’m back in tomorrow.